requirements.in

```diff
@@ -29,7 +30,7 @@ Sphinx==7.4.7
 # https://sphinx-rtd-theme.readthedocs.io/en/stable/changelog.html
 sphinx-rtd-theme==3.0.2
 # https://uwsgi-docs.readthedocs.io/en/latest/#release-notes
-uWSGI==2.0.28
+#uWSGI==2.0.28
```

Comment:
Did you work on windows and had to comment uWSGI because it fails to install? :)
Please uncomment.